ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/ChainSync/Server.hs

```diff
@@ -156,4 +163,8 @@ chainSyncServerForReader tracer chainDB rdr =
 data TraceChainSyncServerEvent blk
   = TraceChainSyncServerRead        (Tip blk) (ChainUpdate blk (Point blk))
   | TraceChainSyncServerReadBlocked (Tip blk) (ChainUpdate blk (Point blk))
+  | TraceChainSyncRollForward
+  | TraceChainSyncRollBackward
```

Comment:
Are we also planning metrics for the rest of these? I only see the `RollForward` event being used downstream.

Reply:
I only intend to use `RollForward` for metrics at the moment.  Would you rather I delete the others?